packages/design-system/src/components/modal/index.js

```diff
@@ -84,6 +84,7 @@ export function Modal({
   overlayStyles,
   ...rest
 }) {
+  // TODO, refactor to use useContext from use-context-selector.
```

Comment:
Still need to remove that TODO. As it's not possible to use `useContext` from `use-context-selector`.